packages/eslint-plugin/src/rules/no-misused-spread.ts

```diff
@@ -64,6 +71,8 @@ export default createRule<Options, MessageIds>({
         'Consider using `Intl.Segmenter` for locale-aware string decomposition.',
         "Otherwise, if you don't need to preserve emojis or other non-Ascii characters, disable this lint rule on this line or configure the 'allow' rule option.",
       ].join('\n'),
+      replaceMapSpreadInObject:
+        'Replace map spread in object with `Object.fromEntries(map)`',
```

Comment:
[Docs] Nit: `map` might be a name of another variable in scope. Using it here could be confusing.

```suggestion
        'Replace map spread in object with `Object.fromEntries()`',
```